db/operations.go

```diff
@@ -51,6 +53,7 @@ func (l *Loader) newUpdateOperation(table *TableInfo, primaryKey map[string]stri
 		opType:     OperationTypeUpdate,
 		primaryKey: primaryKey,
 		data:       data,
+		dialect:    l.getDialect(),
```

Comment:
I think I prefer that we don't put dialect on the operation directly. I think `func (o *Operation) query()` should receive the dialect.

Reply:
Thank you for your suggestion, I wasn't liking this solution either.